yarn-project/circuits.js/package.json

```diff
@@ -40,6 +40,7 @@
   },
   "dependencies": {
     "@aztec/foundation": "workspace:^",
+    "@aztec/merkle-tree": "workspace:^",
```

Comment:
Make sure to run `yarn prepare` in the root of yarn-projects if you change dependencies, so these get reflected in the build manifest

Reply:
dep removed